brainglobe_atlasapi/atlas_generation/wrapup.py

```diff
@@ -196,32 +197,31 @@ def wrapup_atlas_from_data(
 
         # Reorient points:
         mesh.points = space_convention.map_points_to(
-            descriptors.ATLAS_ORIENTATION, mesh.points
+            TARGET_ORIENTATION, mesh.points
         )
 
         # Save in meshes dir:
         mio.write(mesh_dest_dir / f"{mesh_id}.obj", mesh)
 
-    transformation_mat = space_convention.transformation_matrix_to(
-        descriptors.ATLAS_ORIENTATION
-    )
-
     # save regions list json:
     with open(dest_dir / descriptors.STRUCTURES_FILENAME, "w") as f:
         json.dump(structures_list, f)
 
+    final_resolution = space_convention.map_resolution(
+        TARGET_ORIENTATION, resolution
+    )
+
```

Comment:
@alessandrofelder you've been packaging atlases much more than me recently. Do we have any convention about whether the resolution is passed from the packaging scripts in the original orientation or `asr`? My gut feeling is that this hasn't actually come up yet (the only anisotropic atlas is already in ASR I think). 

Basically, do you think this change should stay? My gut feeling is yes (and it should always have been there), but I'm not 100%

Reply:
I think @viktorpm and I had this problem before in #322 and solved it in a less elegant way than suggested here, by adding an optional `resolution_mapping` parameter to the `wrapup_atlas_from_data` function. 

We shouldn't have two ways to remap the resolution, but I am happy for you and @thisisrick25 to choose which one to keep. The one here is more elegant, but if we keep it we should probably adapt the AMBDA script accordingly, as it could be very confusing if we ever repackaged AMBDA?

Reply:
Good knowledge. I think we can keep the way in this PR, and I will raise an issue for the AMBDA.